src/style.css

```diff
@@ -1 +1,36 @@
-/* add styles here */
+* {
+  margin: 0;
+  padding: 0;
+}
```

Comment:
Avoid using the universal selector '*' for setting margin and padding to zero as it impacts performance. It's better to set styles only for the elements that require them.